docs/swagger/spec.yml

```diff
@@ -611,38 +655,225 @@ paths:
           description: Thumbnail UUID was not found
           schema:
             $ref: '#/definitions/Error'
-  /boundaries/:
+  /tokens/:
     get:
-      summary: Paginated list of boundaries
+      summary: |
+        Retrieve a list of refresh token summaries. Note: Only names and identifiers are revealed,
+        refresh tokens themselves are only revealed once, when created for security purposes.
       tags:
-        - Imagery
+        - Authentication
+      responses:
+        200:
+          description: List of refresh token identifiers
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/RefreshToken'
+        404:
+          description: No refresh tokens found for user
+          schema:
+            $ref: '#/definitions/Error'
+    post:
+      summary: |
+        Request a new authorization token to use in requests
+      tags:
+        - Authentication
+      responses:
+        201:
+          description: Created refresh token object
+          schema:
+            $ref: '#/definitions/AuthToken'
+        default:
+          description: Error message if insufficient permissions are present
+  /tokens/{refreshTokenId}/:
+    delete:
+      parameters:
+        - name: refreshTokenId
+          in: path
+          required: true
+          type: string
+      summary: Delete a refresh token and revoke its access
+      tags:
+        - Authentication
+      responses:
+        204:
+          description: Revocation successful, No Content
```

Comment:
Fixed in [1646ba2](https://github.com/azavea/raster-foundry/pull/613/commits/1646ba2e46f2f1831e32fade34773196bba55255)